packages/unbound/build.sh

```diff
@@ -3,9 +3,10 @@ TERMUX_PKG_DESCRIPTION="A validating, recursive, caching DNS resolver"
 TERMUX_PKG_LICENSE="BSD 3-Clause"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1.15.0
+TERMUX_PKG_REVISION=1
 TERMUX_PKG_SRCURL=https://nlnetlabs.nl/downloads/unbound/unbound-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=a480dc6c8937447b98d161fe911ffc76cfaffa2da18788781314e81339f1126f
-TERMUX_PKG_DEPENDS="libevent, libexpat, libnghttp2, openssl, python"
+TERMUX_PKG_DEPENDS="libevent, libexpat, libnghttp2, openssl"
```

Comment:
`lib/libunbound.so` links against `libpython3.10.so.1.0`. `python` cannot be removed from `TERMUX_PKG_DEPENDS`.

Reply:
Thanks, yeah, this won't work then